src/emitter/json.rs

```diff
@@ -120,6 +109,9 @@ mod tests {
 
     #[test]
     fn expected_line_range_correct_when_single_line_split() {
+        let mut emitter = JsonEmitter {
```

Comment:
Nit: we can use `JsonEmitter::default()`.

Reply:
Good catch. I need to make a similar update to the checkstyle xml emitter at some point (which suffers from the same type of invalid output in this scenario) and will tweak this accordingly when I do